### ISSUE TITLE
a11y: add component PhoneNumber Control

### ISSUE THREAD

**Issue by sonya0504** [infermedica/component-library#314]
Please add to CL component with to fields: dropdown with suggested country code and input to enter user phone number.

Please take care about a11y. After finish work, check component using Wave plugin and solve this a11y problems:

- 1 X Missing form label (error)
- 1 X Orphaned form label (alert)

### Screenshots
![image](https://github.com/infermedica/component-library/assets/37744371/ddd2f628-eef3-4abc-8534-423c59334049)

**Comment by pspaczek** [comment]
added in #323
